### ISSUE TITLE
Can't minimize if only one window is open

### ISSUE THREAD

**Issue by Razer0123** [pop-os/shell#1345]
**(1) Issue/Bug Description:**

Can't hide / minimize when only one window is active

**(2) Steps to reproduce (if you know):**

Minimize all windows but one

**(3) Expected behavior:**

In the past i was able to hide all windows and see the desktop

**(4) Distribution (run `cat /etc/os-release`):**
 
```
NAME="Fedora Linux"
VERSION="35 (Workstation Edition)"
ID=fedora
VERSION_ID=35
VERSION_CODENAME=""
PLATFORM_ID="platform:f35"
PRETTY_NAME="Fedora Linux 35 (Workstation Edition)"
ANSI_COLOR="0;38;2;60;110;180"
LOGO=fedora-logo-icon
CPE_NAME="cpe:/o:fedoraproject:fedora:35"
HOME_URL="https://fedoraproject.org/"
DOCUMENTATION_URL="https://docs.fedoraproject.org/en-US/fedora/f35/system-administrators-guide/"
SUPPORT_URL="https://ask.fedoraproject.org/"
BUG_REPORT_URL="https://bugzilla.redhat.com/"
REDHAT_BUGZILLA_PRODUCT="Fedora"
REDHAT_BUGZILLA_PRODUCT_VERSION=35
REDHAT_SUPPORT_PRODUCT="Fedora"
REDHAT_SUPPORT_PRODUCT_VERSION=35
PRIVACY_POLICY_URL="https://fedoraproject.org/wiki/Legal:PrivacyPolicy"
VARIANT="Workstation Edition"
VARIANT_ID=workstation
```

**(5) Gnome Shell version:**

41.3

**(6) Pop Shell version (run `apt policy pop-shell` or provide the latest commit if building locally):**

Not sure, package is 1.2.0^5.21745c4-1.fc35

**(7) Where was Pop Shell installed from:**

Fedora repository

**(8) Monitor Setup (2 x 1080p, 4K, Primary(Horizontal), Secondary(Vertical), etc):**
1 x 3440x1440

**(9) Other Installed/Enabled Extensions:**

**(10) Other Notes:**

**Comment by davidhoang05022009** [comment]
Same here, on 1 x 1920x1080 monitor

**Comment by Razer0123** [comment]
Weird nobody else noticed this

**Comment by CarmineTesta** [comment]
Same for me.

**Comment by davidhoang05022009** [comment]
So I switched to xorg but still having this problem. I'm going to rollback to 1.1.0

**Comment by davidhoang05022009** [comment]
OK, I fixed this by downgrading to 1.2.0^4.1fddaa8-2, now I can minimize when only one window is open.
Steps to downgrade on Fedora:
- Go to Koji and downloaded these files 
- [gnome-shell-extension-pop-shell-1.2.0^4.1fddaa8-2.fc36.noarch.rpm](https://kojipkgs.fedoraproject.org//packages/gnome-shell-extension-pop-shell/1.2.0%5E4.1fddaa8/2.fc36/noarch/gnome-shell-extension-pop-shell-1.2.0%5E4.1fddaa8-2.fc36.noarch.rpm) 
- [gnome-shell-extension-pop-shell-shortcut-overrides-1.2.0^4.1fddaa8-2.fc36.noarch.rpm
](https://kojipkgs.fedoraproject.org//packages/gnome-shell-extension-pop-shell/1.2.0%5E4.1fddaa8/2.fc36/noarch/gnome-shell-extension-pop-shell-shortcut-overrides-1.2.0%5E4.1fddaa8-2.fc36.noarch.rpm)
- Downgrade by using dnf: 
```sh
sudo dnf downgrade /path/to/gnome-shell-extension-pop-shell-1.2.0^4.1fddaa8-2.fc36.noarch.rpm -y
sudo dnf downgrade /path/to/gnome-shell-extension-pop-shell-shortcut-overrides-1.2.0^4.1fddaa8-2.fc36.noarch.rpm -y
```

**Comment by Razer0123** [comment]
> OK, I fixed this by downgrading to 1.2.0^4.1fddaa8-2, now I can minimize when only one window is open. Steps to downgrade on Fedora:
> 
>     * Go to Koji and downloaded these files
> 
>     * [gnome-shell-extension-pop-shell-1.2.0^4.1fddaa8-2.fc36.noarch.rpm](https://kojipkgs.fedoraproject.org//packages/gnome-shell-extension-pop-shell/1.2.0%5E4.1fddaa8/2.fc36/noarch/gnome-shell-extension-pop-shell-1.2.0%5E4.1fddaa8-2.fc36.noarch.rpm)
> 
>     * [gnome-shell-extension-pop-shell-shortcut-overrides-1.2.0^4.1fddaa8-2.fc36.noarch.rpm
>       ](https://kojipkgs.fedoraproject.org//packages/gnome-shell-extension-pop-shell/1.2.0%5E4.1fddaa8/2.fc36/noarch/gnome-shell-extension-pop-shell-shortcut-overrides-1.2.0%5E4.1fddaa8-2.fc36.noarch.rpm)
> 
>     * Downgrade by using dnf:
> 
> 
> ```shell
> sudo dnf downgrade /path/to/gnome-shell-extension-pop-shell-1.2.0^4.1fddaa8-2.fc36.noarch.rpm -y
> sudo dnf downgrade /path/to/gnome-shell-extension-pop-shell-shortcut-overrides-1.2.0^4.1fddaa8-2.fc36.noarch.rpm -y
> ```

thanks for posting this workaround, but i hope they will fix this upstream

**Comment by ragnarokxg** [comment]
Having this same issue, and have downgraded until the issue is resolved.  My son who uses PopOS is able to minimize all windows so this is definitely a bug and not a feature.

**Comment by davidhoang05022009** [comment]
Just switched to Material Shell because of this, the active hint issue with kitty and flickering when resizing windows (so annoying but still love Pop Shell)

**Comment by jacobgkau** [comment]
I can recreate this in a Fedora 35 live disk with GNOME 41 on Wayland. (Tiling does not work on Fedora 35 in X11.) It is not happening in Pop!_OS 21.10 (GNOME Shell 40.5), but it is happening on Pop!_OS 22.04 beta (GNOME Shell 41.3). That means you can expect this to be solved before Pop!_OS 22.04 LTS is released.

> i am certain this is also an easy fix

@Razer0123 If you are certain the fix is easy, feel free to provide details about what the fix would be. If you don't have details to back up that assertion, then you may want to refrain from baseless speculation.
